bigbone/src/main/kotlin/social/bigbone/api/method/NotificationMethods.kt

```diff
@@ -13,16 +13,21 @@ import social.bigbone.api.exception.BigBoneRequestException
  */
 class NotificationMethods(private val client: MastodonClient) {
```

Comment:
Kdoc for the class references an alternative "api/vX/notification" (singular) endpoint, which was removed by code changes and apparently never existed, assuming a Mastodon doc error at the time of first implementing this. Please remove that reference.

Reply:
Done in c5d3329